app/account/hooks/useUsersUpdate.ts

```diff
@@ -0,0 +1,56 @@
+import { useToast } from '@chakra-ui/react'
+import { useRouter } from 'next/navigation'
+import { useUserId } from '@/providers/session-provider'
+import { useUserGet } from '.'
+import { useUpdateUserMutation } from '@generated/api'
+
+export const useUserUpdate = () => {
+  const userId = useUserId()
+  const router = useRouter()
+  const toast = useToast()
+  const { userRefetch } = useUserGet()
+
+  const [updateUserMutation, { loading: isLoading }] = useUpdateUserMutation()
+
+  const updateUser = async (name: string, email: string) => {
```

Comment:
I tried to log in with a new email address after I updated the email address and logged out of the account. But I couldn't. Could you check it?